nitro-tag.txt

```diff
@@ -0,0 +1 @@
+dev
```

Comment:
I might've missed something, but where is this used?

Reply:
The issue is that if a dev build was already created at a given commit, Docker caches that image.  I added code to the AWS CodeBuild process to save the current version tag into nitro-tag.txt, that way if a relase tag is added, the docker build is appropriately rebuilt since the file is copied into the docker image.

Reply:
Do we need to commit this to git though, or can AWS just set it?

Reply:
Figured out workaround to prevent `docker build` from failing when file doesn't exist.

Reply:
workaround fixed docker, but messed up docker build caching in CI, so reverted to using `.nitro-tag.txt` instead